infer_unet_widget.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (C) 2021 Ikomia SAS
+# Contact: https://www.ikomia.com
+#
+# This file is part of the IkomiaStudio software.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from ikomia import core, dataprocess
+from ikomia.utils import pyqtutils, qtconversion
+from infer_unet.infer_unet_process import InferUnetParam
+
+# PyQt GUI framework
+from PyQt5.QtWidgets import *
+
+
+# --------------------
+# - Class which implements widget associated with the process
+# - Inherits PyCore.CWorkflowTaskWidget from Ikomia API
+# --------------------
+class InferUnetWidget(core.CWorkflowTaskWidget):
+
+    def __init__(self, param, parent):
+        core.CWorkflowTaskWidget.__init__(self, parent)
+
+        if param is None:
+            self.parameters = InferUnetParam()
+        else:
+            self.parameters = param
+
+        # Create layout : QGridLayout by default
+        self.gridLayout = QGridLayout()
+        # PyQt -> Qt wrapping
+        layout = qtconversion.PyQtToQt(self.gridLayout)
+
+        img_scaleLabel = QLabel("Scale (resize images)")
+        self.img_scaleSpinBox = QDoubleSpinBox()
```

Comment:
You can use PyQtUtils module of Ikomia API to create basic Qt components in one line.